packages/react-components/react-tree/src/components/TreeItemActions/TreeItemActions.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
```

Comment:
to reduce the size of this PR you could follow up with TreeItemsActions after the big change of introducing leaft/branch item types

Reply:
Ok, I tried to split the restructuring and the `TreeItemActions` component into 2 PR. So here's the PR with just the restructuring: https://github.com/microsoft/fluentui/pull/27799 (it's still quite huge though 😅)